cairo/ethereum/cancun/state.cairo

```diff
@@ -685,8 +684,7 @@ func close_transaction{
     with_attr error_message("IndexError") {
         tempvar parent_main_trie_ptr = cast(parent_main_trie, felt);
         if (cast(parent_main_trie_ptr, felt) == 0) {
-            assert 0 = 1;
-            ret;
+            raise('IndexError');
```

Comment:
what happens in this case? it looks like there are two with_attr error_message nested, the one in line 684 and the one in raise

Reply:
it's possible to have several nested `with_attr` , but here it's just forgotten to be removed